mullvad-cli/src/format.rs

```diff
@@ -225,6 +225,17 @@ fn print_error_state(error_state: &ErrorState) {
             println!("Blocked: {cause}");
             println!("Your kernel might be terribly out of date or missing nftables");
         }
+        #[cfg(target_os = "macos")]
+        cause @ talpid_types::tunnel::ErrorStateCause::NeedFullDiskPermissions => {
+            println!("Blocked: {cause}");
+            println!();
+            println!(r#"Enable "Full Disk Access" for "Mullvad VPN" in the macOS system settings:"#);
+            println!(r#"open "x-apple.systempreferences:com.apple.preference.security?Privacy_AllFiles"#);
```

Comment:
Is this formatted correctly? Seems like there is a `"` before `x-apple`, but not ending quote.

Reply:
You're right! Fixed.